hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestHFile.java

```diff
@@ -890,4 +892,89 @@ public void testDBEShipped() throws IOException {
       writer.close();
     }
   }
+
+  /**
+   * Test case for CombinedBlockCache with TinyLfu as L1 cache
+   */
+  @Test
+  public void testReaderWithTinyLfuCombinedBlockCache() throws Exception {
```

Comment:
These 2 are same tests cases but only diff is L1 policy name? If so can we avoid duplication and have single private method and call with diff args?

Reply:
Done